### PR TITLE
Add the meetup0726 file

### PR DESCRIPTION
**Pull request by kuomc**

### CODE REVIEW

File: content/pages/meetup/2023/0726-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    * Task arrangement for volunteers
+    * Promotion plan for August scisprint
+    * Preparations for career conversation
+
+|
+
+Sign up
+------------
+
+The event is free. Please register on `sign up <https://sciwork.kktix.cc/events/meetup-20230726>`__.
```
Comment (YenPeiChen07) [inline content/pages/meetup/2023/0726-nycu.rst:48]: The text could be written as `The event is free. Please register at kktix` ,which may be more clear.

File: content/pages/meetup/2023/0726-nycu.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Meetup is an event providing space for people to work on open source
+projects together. We welcome any subjects that may interest the attendees,
+and especially encourage code for science, engineering, and technology, which
+demand more critical discussions than other applications of computer
+programming.
+
+We would like to provide a supportive and friendly environment for all 
+attendees to support more developers to join in the open-source communities. 
+
+To join the meetup, please bring your laptop. You are also very welcome to
```
Comment (YenPeiChen07) [inline content/pages/meetup/2023/0726-nycu.rst:64]: Please add some text tot tell the attendees to sign up at kktix.

You can reference in [scisprint May webpage](https://github.com/sciwork/swportal/blob/a74570cf32bc5cd6b0d4341e83ee003333821d65/content/pages/sprint/2023/05-hsinchu.rst?plain=1#L30).

**Review by YenPeiChen07 — CHANGES_REQUESTED** [review]

File: content/pages/meetup/2023/0726-nycu.rst (hunk 1 above)
Comment (yungyuc) [reply content/pages/meetup/2023/0726-nycu.rst]: Agree, something like:

```rst
The event is free. Please register at `kktix <https://sciwork.kktix.cc/events/meetup-20230726>`__.
```

File: content/pages/meetup/2023/0726-nycu.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    * Task arrangement for volunteers
+    * Promotion plan for August scisprint
+    * Preparations for career conversation
+
+|
+
+Sign up
+------------
+
+The event is free. Please register on `kktix <https://sciwork.kktix.cc/events/meetup-20230726>`__.
```
Comment (yungyuc) [inline content/pages/meetup/2023/0726-nycu.rst:48]: Either "on" or "at" is fine.

**Review by yungyuc — APPROVED** [review]

**Review by YenPeiChen07 — APPROVED** [review]

**Pull request merged**